### PR TITLE
Sync firmware-imx_8.23.bb with meta-imx-bsp layer

### PR DESCRIPTION
**Pull request by Livius90**
Sync firmware-imx_8.23 recipe with meta-imx-bsp layer.

### CODE REVIEW

File: recipes-bsp/firmware-imx/firmware-imx_8.23.bb [hunk 1]
```diff
@@ -16,8 +16,9 @@ do_install() {
     # SDMA Firmware section
     install -d ${D}${nonarch_base_libdir}/firmware/imx/sdma
     install -m 0644 ${S}/firmware/sdma/* ${D}${nonarch_base_libdir}/firmware/imx/sdma
-    rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx6q.bin
-    rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx7d.bin
+    # Comment these lines to use sdma-imx6q/7d.bin from here and not linux-firmware
+    #rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx6q.bin
+    #rm -f ${D}${nonarch_base_libdir}/firmware/imx/sdma/sdma-imx7d.bin
```
Comment (otavio) [inline recipes-bsp/firmware-imx/firmware-imx_8.23.bb:21]: Please let our change in this hook.

File: recipes-bsp/firmware-imx/firmware-imx_8.23.bb [hunk 2]
```diff
@@ -144,13 +151,15 @@ PACKAGES_DYNAMIC = "${PN}-vpu-* ${PN}-sdma-* ${PN}-easrc-* ${PN}-xcvr-* ${PN}-xu
...
-PACKAGES = "${PN} ${PN}-epdc ${PN}-hdmi"
+PACKAGES = "${PN} ${PN}-epdc ${PN}-hdmi ${PN}-vpu-imx8 ${PN}-vpu-imx95"
 
 FILES:${PN}-epdc = "${nonarch_base_libdir}/firmware/imx/epdc/"
 FILES:${PN}-hdmi = " \
     ${nonarch_base_libdir}/firmware/hdmitxfw.bin \
     ${nonarch_base_libdir}/firmware/hdmirxfw.bin \
     ${nonarch_base_libdir}/firmware/dpfw.bin \
 "
+FILES:${PN}-vpu-imx8 = "${nonarch_base_libdir}/firmware/amphion/vpu/*"
```
Comment (otavio) [inline recipes-bsp/firmware-imx/firmware-imx_8.23.bb:162]: Use `vpu-amphion` as suffix.

File: recipes-bsp/firmware-imx/firmware-imx_8.23.bb [hunk 3]
```diff
@@ -144,13 +151,15 @@ PACKAGES_DYNAMIC = "${PN}-vpu-* ${PN}-sdma-* ${PN}-easrc-* ${PN}-xcvr-* ${PN}-xu
...
+FILES:${PN}-vpu-imx95 = "${nonarch_base_libdir}/firmware/wave633c_codec_fw.bin"
```
Comment (otavio) [inline recipes-bsp/firmware-imx/firmware-imx_8.23.bb:163]: Use `vpu-wave` as suffix.

**Review by otavio — CHANGES_REQUESTED** [review]

**Review by MrCry0 — APPROVED** [review]

File: recipes-bsp/firmware-imx/firmware-imx_8.23.bb (hunk 2 above)
Comment (Livius90) [reply recipes-bsp/firmware-imx/firmware-imx_8.23.bb:162]: Do you mean `FILES:${PN}-vpu-amphion-imx8`?

File: recipes-bsp/firmware-imx/firmware-imx_8.23.bb (hunk 3 above)
Comment (Livius90) [reply recipes-bsp/firmware-imx/firmware-imx_8.23.bb:163]: Do you mean the following?
```
FILES:${PN}-vpu-wave-imx8 = "${nonarch_base_libdir}/firmware/amphion/vpu/*"
FILES:${PN}-vpu-wave-imx95 = "${nonarch_base_libdir}/firmware/wave633c_codec_fw.bin"
```

File: recipes-bsp/firmware-imx/firmware-imx_8.23.bb (hunk 1 above)
Comment (Livius90) [reply recipes-bsp/firmware-imx/firmware-imx_8.23.bb:21]: If `rm` is used for sdma FWs, they are never be able to use from official firmware-imx recipe, why is it good?

Comment (otavio) [reply recipes-bsp/firmware-imx/firmware-imx_8.23.bb:21]: We use the `linux-firmware` provided ones.

File: recipes-bsp/firmware-imx/firmware-imx_8.23.bb (hunk 2 above)
Comment (otavio) [reply recipes-bsp/firmware-imx/firmware-imx_8.23.bb:162]: I'd not use imx8 suffix as there is no restriction for NXP to make another SoC with Amphion VPU

**Pull request merged**